_1327/documents/templates/documents_edit.html

```diff
@@ -372,7 +372,7 @@ <h4 class="modal-title" id="deleteAutosaveHeader">{% trans "Do you really want t
 	// init emoji stuff
 	$('#id_text').emojioneArea({
 		shortnames: true,
-		saveEmojisAs: 'unicode',
+		saveEmojisAs: 'shortname',
```

Comment:
the diff view doesn't display emojis anymore when the document is edited after this change because they are no longer stored in unicode. why does this help again?

Reply:
Hmm, I hoped this makes it nicer when writing the text, as emojis do not necessarily get translated into the unicode representation directly in the editor...  but if it breaks other things... we should keep it the way it was ;)